Formula/o/openshift-cli.rb

```diff
@@ -1,7 +1,7 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.14/openshift-client-src.tar.gz"
+  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.15/openshift-client-src.tar.gz"
   sha256 "6b6ac6109dbbc8240a343d4a07661cf98ae008305bf432b58ca6b9c00c5a83d5"
```

Comment:
Dismissed review as looks like version number changed in URL but sha256 for tarball is the same.

Reply:
Looking at https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.15/sha256sum.txt. and https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.14/sha256sum.txt it did indeed not change. They maybe just re-release from the same source and made a rebuild of their binaries ...

Anyway  openshift-client-src-4.13.17-x86_64.tar.gz is out and this one has a different sha